### DISCUSSION TITLE
Import Github APi results to Dojo

### DISCUSSION THREAD

**Discussion by vyastj** [DefectDojo/django-DefectDojo#6602] [Q&A]
Hello

I want to import Github's API results to defect dojo having vulnerability data of around 2000 repositories. This doesn't seem to be possible. Any suggestions?

**Comment by damiencarol** [comment]
A parser exists today for Github API. you vulnerabilities are sec advisory?

**Reply by vyastj** [reply]
The vulnerability data is from Github's Depandabot Security alerts in form of `json`. I have got that using Github's Graphql API

**Reply by damiencarol** [reply]
so it should be compatible with one of the parser.

**Reply by vyastj** [reply]
Yes, I am able to parse results for limited data (1-2 repositories), but when it comes to importing data to Dojo for the whole organization, it doesnt work

**Reply by damiencarol** [reply]
could you discuss it in Slack? better to discuss. => https://owasp-slack.herokuapp.com/

**Reply by vyastj** [reply]
yep, sure
